scripts/tools/silabs/ota/ota_multi_image_tool.py

```diff
@@ -312,6 +342,8 @@ def any_base_int(s): return int(s, 0)
 
     create_parser.add_argument('-app', "--app-input-file",
                                help='Path to application input file')
+    create_parser.add_argument('-wifi', "--wifi-input-file",
+                               help='Path to OTA image for SiWx917 (TA/M4/Combined file), 917 NCP (TA)')
```

Comment:
If we don't need a specific tag, we don't need this change either.

Reply:
This wifi_ta is needed for 917 NCP OTA upgrade case.